interface/tuntap/TUNInterface_freebsd.c

```diff
@@ -46,10 +46,19 @@ struct Iface* TUNInterface_new(const char* interfaceName,
                                    struct Except* eh,
                                    struct Allocator* alloc)
 {
+    char deviceFile[TUNInterface_IFNAMSIZ];
+
     if (isTapMode) { Except_throw(eh, "tap mode not supported on this platform"); }
 
+    // We are on FreeBSD so we just need to read /dev/tunxx to create the tun interface
+    if (interfaceName) {
+        sprintf(deviceFile,"/dev/%s",interfaceName);
```

Comment:
Did you check that the interface name doesn't smash the stack? #FreeNSD  #90sBugs